src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
```

Comment:
Consider adding validation to the `email` property to ensure it is a valid email address. 📧